src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -66,12 +66,8 @@ public class ConvertFromMarkdownCommand : PSCmdlet
         /// </summary>
         protected override void BeginProcessing()
         {
-            _mdOption = this.CommandInfo.Module.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
-
-            if (_mdOption == null)
-            {
-                throw new InvalidOperationException();
-            }
+            _mdOption = this.CommandInfo.Module?.SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
```

Comment:
Could you please clarify for my education why is `this.CommandInfo.Module` null in the scenario?

Reply:
Its part of the PR context.

Reply:
My question is why this is null if it is running in new runspace. I do not understand the difference and therefore I am afraid that there may be a bug.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.commandinfo.module?view=powershellsdk-1.1.0#System_Management_Automation_CommandInfo_Module indicates that this shouldn't be null since this cmdlet is part of a module.  Seems like there's a bug somewhere else.  I don't think this would block this PR, but we should have an issue open to investigate this.

Reply:
We already have #9390 for tracking. If it is "a bug somewhere else". Since we are at the beginning of a new milestone we could only add pending test in the PR without temporary fix.

Reply:
My assumption of it being because of snapins seems to be wrong. @daxian-dbw might have some insights here.

Reply:
Yes, if it's a snapin, it won't show up as a module

Reply:
Yep, it's because powershell built-in modules are loaded as snapins with `[runspacefactory]::CreateRunspacePool(1, 2, $Host)`, as well as `[runspacefactory]::CreateRunspace()`.
This issue repros with the following script too:
```
PS:100> $ps = [powershell]::Create()
PS:101> $ps.AddScript("'# test' | convertfrom-markdown").Invoke()
PS:102> $ps.Streams.Error
convertfrom-markdown : Object reference not set to an instance of an object.
At line:1 char:12
+ '# test' | convertfrom-markdown
+            ~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [ConvertFrom-Markdown], NullReferenceException
+ FullyQualifiedErrorId : System.NullReferenceException,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand
```

Reply:
Thanks for clarify. If snapins is deprecated should we change loading built-in modules too? If possible this would be the best fix.

Reply:
The public interface of PSSnapin is deprecated, meaning user cannot define a snapin to work with PowerShell, but the snapin itself is still useful, especially in the hosting scenario.
Loading as snapin doesn't require the modules to be present, so for some hosting scenario, this is convenient as the hosting application doesn't need to ship the built-in powershell modules along with the application.

Reply:
I wonder - we can create an assembly with Add-Type and load it with Import-Module. We use this in our tests without creating module folder and etc.